### PR TITLE
Fix testcase

### PR DESCRIPTION
**Pull request by JunhwanPark**
I have tested the tc examples of the kernel and network symbols. This patch includes examples that fix build errors and correspond to modified kernels.

### CODE REVIEW

File: apps/examples/testcase/le_tc/kernel/tc_libc_timer.c [hunk 1]
```diff
@@ -378,15 +378,15 @@ static void tc_libc_timer_clock_daysbeforemonth(void)
 	for (month_iter = 1; month_iter < 12; month_iter++) {
 		prev_month = clock_daysbeforemonth(month_iter, FALSE);
 		cur_month = clock_daysbeforemonth(month_iter+1, FALSE);
-		TC_ASSERT_EQ("clock_daysbeforemonth", cur_month-prev_month, notleapyear_days[month_iter]);
+		TC_ASSERT_EQ("clock_daysbeforemonth", cur_month-prev_month, notleapyear_days[month_iter+1]);
```
Comment (heesub) [inline apps/examples/testcase/le_tc/kernel/tc_libc_timer.c]: You need to put a space before and after '+' operator, though I do not like TizenRT coding style.

File: apps/examples/testcase/le_tc/kernel/tc_libc_timer.c [hunk 2]
```diff
@@ -378,15 +378,15 @@ static void tc_libc_timer_clock_daysbeforemonth(void)
...
 	}
 
 	/* test with leapyear */
 
 	for (month_iter = 1; month_iter < 12; month_iter++) {
 		prev_month = clock_daysbeforemonth(month_iter, TRUE);
 		cur_month = clock_daysbeforemonth(month_iter+1, TRUE);
-		TC_ASSERT_EQ("clock_daysbeforemonth", cur_month-prev_month, leapyear_days[month_iter]);
+		TC_ASSERT_EQ("clock_daysbeforemonth", cur_month-prev_month, leapyear_days[month_iter+1]);
```
Comment (heesub) [inline apps/examples/testcase/le_tc/kernel/tc_libc_timer.c]: ditto

File: apps/examples/testcase/le_tc/kernel/tc_libc_timer.c (hunk 1 above)
Comment (JunhwanPark) [inline apps/examples/testcase/le_tc/kernel/tc_libc_timer.c]: u r right! I'll fix it. Thanx

**Comment by heesub** [comment]
(long sigh) I've noticed that even the original code does not follow the damn TizenRT coding style.

You need to squash the third commit into the first one.

**Comment by heesub** [comment]
And one more thing, I never understand the purpose of testcase. Why does it check the equality between `cur_month-prev_month` and `leapyear_days[month_iter]`? Meaningless.

**Comment by JunhwanPark** [comment]
Hi, @junmin-kim 
Can you give a comment on the `tc_libc_timer_clock_daysbeforemonth` symbol? You were the last person to change this file.

**Comment by JunhwanPark** [comment]
@heesub 
haha, OK. You are right. The other code is also `month_iter+1`, so I changed it.
`cur_month = clock_daysbeforemonth(month_iter+1, FALSE);`

**Comment by junmin-kim** [comment]
@JunhwanPark I agree with "month_iter+1" is right.

**Comment by JunhwanPark** [comment]
@junmin-kim 
Thanks for your reply. I want answers to questions from @heesub . Did not you develop the TC source?

**Comment by junmin-kim** [comment]
@JunhwanPark and, I think month_iter should be init to zero.

**Comment by junmin-kim** [comment]
@JunhwanPark How about this? Isn't it look better?
`	int notleapyear_days[12] = { 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 };

	/* test with not leapyear */

	for (month_iter = 1; month_iter < 12; month_iter++) {
		prev_month = clock_daysbeforemonth(month_iter-1, FALSE);
		cur_month = clock_daysbeforemonth(month_iter, FALSE);
		TC_ASSERT_EQ("clock_daysbeforemonth", cur_month-prev_month, notleapyear_days[month_iter]);
	}`

**Comment by JunhwanPark** [comment]
@junmin-kim 
Nice. I think the way you gave me is good. This matches the variable name meaning of prev_month.

**Comment by sunghan-chang** [comment]
If you use Junmin's code, you must add spaces around '-'

**Comment by JunhwanPark** [comment]
@sunghan-chang Thanks. I will do that.

**Comment by sunghan-chang** [comment]
Do you need to make a new commit for same changes? There are two "example/testcase: fix daysbeforemonth tc". and typo on latest commit (das -> days)

**Comment by JunhwanPark** [comment]
@sunghan-chang Thanks for info. OK. I'll make 2 patch files.

**Comment by JunhwanPark** [comment]
@sunghan-chang I updated patches. (2 patches) Thanks for waiting.

**Pull request merged**